src/components/form/FormTime.js

```diff
@@ -0,0 +1,48 @@
+import React, {useState} from 'react'
+import './Form.css';
+
+const FormTime = ({ date, setDate, time, setTime, allDayEvent, setAllDayEvent, eventName, setEventName, incrementForm}) => {
```

Comment:
This is kinda a future iteration thing, but do you think we can refactor to reuse one of these components?